fizz_buzz_versions/fizz_buzz_exception.dart

```diff
@@ -1,23 +1,28 @@
-void main() {
-  for (int i = 1; i <= 100; i++) {
-    bool fizz = i % 3 == 0;
-
-    bool buzz = i % 5 == 0;
-
-    switch ((fizz, buzz)) {
-      case (true, true):
-        print('FizzBuzz');
-        break;
-      case (true, false):
-        print('Fizz');
-
-        break;
-      case (false, true):
-        print("Buzz");
-        break;
-      default:
-        print(i);
-        break;
+void main(){
+  for(int i = 1;i <= 100;i++)
+  {
+    try
+    {
+      if(i % 15 == 0) throw 'FizzBuzz';
+
+      if(i % 5 == 0) throw 'Buzz';
+
+      if(i % 3 == 0) throw 'Fizz';
+
+      print(i);
```

Comment:
would be better if in this case you would throw i.toString() as well as in other cases. But ok, let it be